in_toto/models/layout.py

```diff
@@ -248,6 +282,38 @@ def __init__(self, **kwargs):
 
     self.validate()
 
+  def display(self):
+    """
+    <Purpose>
+      Represents a Step by returning a string which contains information pertaining to its member fields
+    <Returns>
+      Step Class fields as strings, 
+      i.e. name (threshold), 
+            pubkeys 
+            expected_command, 
+            expected_materials, 
+            expected_products
+    """
+    pubkeys = "{}".format("\n  ".join(self.pubkeys))
+    expected_command = "{}".format(" ".join(self.expected_command))
+    expected_materials = ""
+    expected_products = ""
+
+    for value in self.expected_materials:
+      expected_materials += " {}\n\t".format(" ".join(value))
+
+    for value in self.expected_products:
+      expected_products += " {}\n\t".format(" ".join(value))
+
+    return (
+      "\n    step name: {} (threshold: {})\n"
+      "    pubkeys: {}\n " 
+      "   expected cmd: {}\n " 
+      "   expected input (materials): \n\t{}\n " 
+      "   expected output (products): \n\t{} \n "
+      .format(self.name, self.threshold, pubkeys, in_toto.util.check_string(expected_command), \
```

Comment:
You don't need to escape the newline inside parentheses.

Reply:
Oops, I must have added this erroneously at some point in time. Removed in the latest changes!